library/src/main/java/com/ironz/binaryprefs/BinaryPreferencesBuilder.java

```diff
@@ -0,0 +1,95 @@
+package com.ironz.binaryprefs;
+
+import android.content.Context;
+
+import com.ironz.binaryprefs.cache.CacheProvider;
+import com.ironz.binaryprefs.cache.ConcurrentCacheProviderImpl;
+import com.ironz.binaryprefs.encryption.AesByteEncryptionImpl;
+import com.ironz.binaryprefs.encryption.ByteEncryption;
+import com.ironz.binaryprefs.events.BroadcastEventBridgeImpl;
+import com.ironz.binaryprefs.events.EventBridge;
+import com.ironz.binaryprefs.exception.ExceptionHandler;
+import com.ironz.binaryprefs.file.adapter.FileAdapter;
+import com.ironz.binaryprefs.file.adapter.NioFileAdapter;
+import com.ironz.binaryprefs.file.directory.AndroidDirectoryProviderImpl;
+import com.ironz.binaryprefs.file.directory.DirectoryProvider;
+import com.ironz.binaryprefs.file.transaction.FileTransaction;
+import com.ironz.binaryprefs.file.transaction.MultiProcessTransactionImpl;
+import com.ironz.binaryprefs.lock.LockFactory;
+import com.ironz.binaryprefs.lock.SimpleLockFactoryImpl;
+import com.ironz.binaryprefs.serialization.SerializerFactory;
+import com.ironz.binaryprefs.serialization.serializer.persistable.Persistable;
+import com.ironz.binaryprefs.serialization.serializer.persistable.PersistableRegistry;
+import com.ironz.binaryprefs.task.ScheduledBackgroundTaskExecutor;
+import com.ironz.binaryprefs.task.TaskExecutor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BinaryPreferencesBuilder {
+
+    private static final String DEFAULT_NAME = "default";
+    private final Context context;
+    private final Map<String, Class<? extends Persistable>> persistableByToken = new HashMap<>();
```

Comment:
Ok, but then i need move loop to the com.ironz.binaryprefs.BinaryPreferencesBuilder#registerPersistableTokens or delete that method.